test/onnx/test_operators.py

```diff
@@ -740,10 +740,6 @@ def test_empty_like(self):
         x = torch.randn(5, 8, requires_grad=True)
         self.assertONNX(lambda x: torch.empty_like(x), x)
 
-    def test_empty_like_opset7(self):
-        x = torch.randn(5, 8, requires_grad=True)
-        self.assertONNX(lambda x: torch.empty_like(x), x, opset_version=7)
```

Comment:
nit: curious why previously checker does not complain about constantFill?

Reply:
I will check how previous checker take this question. Not in a way that we need it alternatively, but to see what makes the difference here. As we already have a test_empty_like for more current opset version, I think the result won't make regression. But I will investigate on this.

Reply:
Supplementary:

The previous checker doesn't check its reliability before process the constant value map, so the inferred shape is static. However, this experimental node is far from reliable as it's removed (https://github.com/onnx/onnx/pull/1434). And in shape inference we assign dynamic shape for unreliable node. 

A doable fix to save this onnx::ConstantFill would be add it here:
https://github.com/pytorch/pytorch/blob/a7299647d3eaf5e152ed03cc278a2b990cf257ad/torch/csrc/jit/passes/onnx/shape_type_inference.cpp#L1855-L1861
Then the test will pass as it stands reliable like onnx::Constant node.